Inventory.cs

```diff
@@ -0,0 +1,81 @@
+﻿using IMS;
+public class Inventory : IInventory
+{
+    public List<Product> Products { get; } = new List<Product>();
+    public static void Main()
+    {
+        IInventory inventory = new Inventory();
+        var productRepository = new ProductRepository(inventory);
```

Comment:
hmm this is not quite right
if you see it in a certain resource please provide it to me
else search more on dependency injection and tell me if you wanna help